pkg/ansible/controller/controller.go

```diff
@@ -95,12 +96,22 @@ func Add(mgr manager.Manager, options Options) *controller.Controller {
 		log.Error(err, "")
 		os.Exit(1)
 	}
+
 	u := &unstructured.Unstructured{}
 	u.SetGroupVersionKind(options.GVK)
+
+	filterPredicate, err := predicate.NewResourceFilterPredicate(options.Selector)
+
+	if err != nil {
+		log.Error(nil, "Unable to retreieve requirement", err)
```

Comment:
The first argument here should be the error, and the message should make it a little more clear why it exited. The user won't know what you mean by requirement, you need to make it clear that it's an issue with the selector they defined